server/src/main/java/sjt/http/server/response/HttpCode.java

```diff
@@ -0,0 +1,17 @@
+package sjt.http.server.response;
+
+public class HttpCode {
+    public static final int HTTP_OK = 200;
+    public static final int HTTP_NOT_FOUND = 404;
+
+    public static String msg(int httpCode) {
+        switch (httpCode) {
+            case HTTP_OK:
+                return " OK";
```

Comment:
공백을 만들어 주는건 HttpCode 클래스의 책임이 아닌 것 같아요. HttpCode 는 "OK" 를 리턴해 주고, 문자열을 조합하는 클래스에서 구분자를 넣어주는 방식은 어떨까요? 😄
```suggestion
                return "OK";
```

Reply:
넵 그게 좋겠어요,,

`Respone.java` 클래스에 아래와 같은 메서드(startLine을 생성해주는..)를 추가해도 괜찮을 것 같습니다 😎

```
private static String startLine(int code) {
        return "HTTP/1.1 " + code + " " + HttpCode.msg(code) + CRLF;
}
```